templates/components/import-component-modal.html

```diff
@@ -32,17 +33,7 @@ <h4 class="modal-title" id="invitation_modal_title">Import OSCAL Component</h4>
     $("#import_loading_spinner").hide();
     m.modal();
   }
-  function fillJSONContent(file) {
-    filecontents = $('#{{import_form.file.auto_id}}').prop('files')[0];
-    var reader = new FileReader();
-    reader.readAsText(filecontents);
-    reader.onload = function(e) {
-        $('#{{import_form.json_content.auto_id}}').val(e.target.result);
-    };
-  }
 
-  function show_spinner() {
-    $("#import_loading_spinner").show();
-  }
```

Comment:
These are still needed by the Component Importer on the Component Library, so they can't be deleted.

Reply:
If it's a matter of not having these duplicated across both the component and project import forms, I think this material should be separated into an extendable base template, because we can't always ensure that both the component importer and project importer will be available on the same page, so we need to have this JS accessible where it's needed.